docs/faq.mdx

```diff
@@ -24,6 +24,14 @@ Head over to [our installation page](/installation-getting-started/what-am-i-ins
 
 ---
 
+## How many resources does Pieces use on your machine?
+
+Due to our commitment to providing a very secure experience by leveraging on-device models, Pieces does require a decent amount of resources; 2GB is fairly standard for what we offer today. We are always working to make Pieces more performant, but this is just due to the nature of our implementation.
```

Comment:
You should mention: 2 GB RAM

Reply:
@shivay-at-pieces bhaiya thanks for the feedback! I've added the "2 GB RAM" clarification. 😊